app/src/main/java/com/kickstarter/ui/activities/CommentsActivity.kt

```diff
@@ -84,11 +86,35 @@ class CommentsActivity :
     }
 
     override fun onReplyButtonClicked(comment: Comment) {
+        startThreadActivity(comment, true)
     }
 
     override fun onFlagButtonClicked(comment: Comment) {
     }
 
     override fun onCommentGuideLinesClicked(comment: Comment) {
     }
+
+    /**
+     * Start the Thread activity with
+     * @param comment the selected comment to reply
+     * @param openKeyboard
+     *      true: he focus needs to be on the composer view and set the keyboard open when open the activity
+     *      false: in case we just need to open the replies screen
+     *
+     * // TODO: Once the viewReplies UI is completed call this method with openKeyboard = false
+     * // TODO: https://kickstarter.atlassian.net/browse/NT-1955
```

Comment:
@sunday-okpoluaefe  once you start working on https://kickstarter.atlassian.net/browse/NT-1955 for opening thread screen please call `startThreadActivity(comment, false)`

Reply:
ok